db/schema.rb

```diff
@@ -45,6 +45,8 @@
     t.string "slug"
     t.string "identifier_uuid", default: "", null: false
     t.integer "path_id"
+    t.boolean "show_on_homepage", default: false, null: false
+    t.string "badge_uri"
```

Comment:
I originally tried to do this with a helper. It mostly worked, but not for every asset. We could change the file names to make it work but I felt it added a subtle dependency between the badge and the SVG name which would be easy to miss. This at least makes it much more obvious.

Reply:
Great idea 🔥